components/tokens/custom-spectrum/custom-large-vars.css

```diff
@@ -21,4 +21,10 @@ governing permissions and limitations under the License.
 
   --spectrum-slider-tick-mark-height: 13px;
   --spectrum-slider-ramp-track-height: 20px;
+
+  --spectrum-colorwheel-path: "M 119 119 m -119 0 a 119 119 0 1 0 238 0 a 119 119 0 1 0 -238 0.2 M 119 119 m -91 0 a 91 91 0 1 0 182 0 a 91 91 0 1 0 -182 0";
+  --spectrum-colorwheel-path-borders: "M 120 120 m -120 0 a 120 120 0 1 0 240 0 a 120 120 0 1 0 -240 0.2 M 120 120 m -90 0 a 90 90 0 1 0 180 0 a 90 90 0 1 0 -180 0";
+  --spectrum-colorwheel-colorarea-container-size: 182px;
+
+  --spectrum-colorhandle-size: 20px;
```

Comment:
We will no longer need the custom token for colorhanlde-size, once the t[okens for colorhandle are added.](https://jira.corp.adobe.com/browse/DNA-1261). Would we want to add a comment here with a TODO to remove once we can use the token?